quill-jdbc/src/main/scala/io/getquill/context/jdbc/Encoders.scala

```diff
@@ -123,7 +123,7 @@ trait ObjectGenericTimeEncoders { self: Encoders =>
   implicit val offseTimeEncoder: Encoder[OffsetTime] =
     encoder(jdbcTypeOfOffsetTime, (index, value, row) =>
       row.setObject(index, value, jdbcTypeOfOffsetTime))
-  implicit val offseDateTimeEncoder: Encoder[OffsetDateTime] =
+  implicit val offsetDateTimeEncoder: Encoder[OffsetDateTime] =
     encoder(jdbcTypeOfOffsetDateTime, (index, value, row) =>
       row.setObject(index, value, jdbcTypeOfOffsetDateTime))
 }
```

Comment:
git, GitHub, and I don't like missing newlines at EOF. However, I try not to change file style when making spelling fixes as it confuses things.